Source/WebCore/platform/network/cocoa/RangeResponseGenerator.mm

```diff
@@ -96,7 +96,11 @@ static ResourceResponse synthesizedResponseForRange(const ResourceResponse& orig
 void RangeResponseGenerator::removeTask(WebCoreNSURLSessionDataTask *task)
 {
     ASSERT(isMainThread());
-    auto* data = m_map.get(task.originalRequest.URL.absoluteString);
+    auto urlString = task.originalRequest.URL.absoluteString;
```

Comment:
Maybe we should do the same early return like the function that is adding the task:
```
if (!originalRequest.URL)
    return;
```

Reply:
Makes sense.